### ISSUE TITLE
Non-ASCII characters showing up in other editors

### ISSUE THREAD

**Issue by chikega** [atg/chocolat-public#1778]
I noticed after saving an HTML file in chocolat that Non-ASCII characters are showing up in other editors. In BBedit, inverted red question marks seem to be replacing carriage returns. In TextMate "<CR>" is actually showing up. These Non-ASCII characters don't show up in Sublime Text, but when I re-save the file in Sublime Text and open the file up in BBedit or Textmate, everything appears normal (no Non-ASCII characters).

**Comment by atg** [comment]
Is it just carriage returns? It could be that Chocolat is detecting your file as CR (Mac OS classic line endings) and converting all the line endings to that. I think I'd have to see the raw data before opening and after saving to tell.

It could also be a file encoding issue.

**Comment by chikega** [comment]
Yes, it's just carriage returns. This is the response I received on the Google Group "BBedit Talk" and it corrected the problem:

"Since such a file almost certainly contains mixed line endings,
please choose Text -> Normalize Line Endings, then save the
file, and it should display correctly everywhere.


Regards,

  Patrick Woolsey
==
Bare Bones Software, Inc.             <http://www.barebones.com/>"
